packages/service-library/src/servicelib/rabbitmq/_client.py

```diff
@@ -129,10 +150,13 @@ async def subscribe(
                     *(queue.bind(exchange, routing_key=topic) for topic in topics)
                 )
 
+            requeue_on_nack: bool = max_retries_upon_error is None
+
             async def _on_message(
                 message: aio_pika.abc.AbstractIncomingMessage,
             ) -> None:
                 async with message.process(requeue=True, ignore_processed=True):
+                    retry_count: int = message.headers.get(_HEADER_X_RETRY_COUNT, 0)  # type: ignore
```

Comment:
why the type ignore here?

Reply:
forget this, pylance was reporting it, but it's not true, mypy wanted it gone